plugins/gs.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: gs
+spec:
+  version: v0.5.0
+  homepage: https://github.com/giantswarm/kubectl-gs
+  shortDescription: Giant Swarm plug-in for kubectl
+  description: |
+    Simplifies creating clusters and node pools via Giant Swarm Kubernetes
+    control planes, as well as installing app catalogs and apps.
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/giantswarm/kubectl-gs/releases/download/v0.5.0/kubectl-gs_0.5.0_darwin_amd64.tar.gz
+    sha256: "40ea6f92f70482e74688798d89af472d2c4c5b11a444c271aa5bf4d1cd322f33"
+    files:
+    - from: kubectl-gs
+      to: .
```

Comment:
please copy out LICENSE as well.

Reply:
You can also just remove the `files` section, as it defaults to `{from: "*", to: "."}` and will copy out everything. AFAICT, this would also install a `readme.md` file, which is okay.

Reply:
Removing the `files` part, so that all three files are copied:

```nohighlight
ls ~/.krew/store/gs/v0.5.0/
LICENSE     README.md   kubectl-gs*
```